jans-linux-setup/jans_setup/templates/scopes.ldif

```diff
@@ -119,6 +119,17 @@ jansScopeTyp: openid
 objectClass: top
 objectClass: jansScope
 
+dn: inum=6D93,ou=scopes,o=jans
+description: Access Evaluation
+displayName: access_evaluation
+inum: 6D93
+jansAttrs: {"spontaneousClientId":"","spontaneousClientScopes":[],"showInConfigurationEndpoint":true}
```

Comment:
Do you reall need `spontaneousClientId`, `spontaneousClientScopes`, and `showInConfigurationEndpoint` here?

Reply:
@nynymike this is default template value. AS reads it but it's not needed for AuthZEN if that was the question.